tests/test_for_utils.py

```diff
@@ -116,3 +119,42 @@ def test_show_versions(capsys):
     assert lines[7].startswith("Package Versions")
     assert lines[8].startswith("================")
     assert any(l.startswith("memote") for l in lines[9:])
+
+
+@pytest.fixture(scope="function")
+def mock_repo(tmpdir_factory):
+    """Mock repo with history: add, modify, do nothing, delete file."""
+    path = tmpdir_factory.mktemp("memote-mock-repo")
```

Comment:
I believe this change should fix the failing tests.

```suggestion
    path = str(tmpdir_factory.mktemp("memote-mock-repo"))
```